pi-scripts/ZotCourt+.py

```diff
@@ -0,0 +1,47 @@
+import RPi.GPIO as GPIO #to control pins!
+import time #for creating delays in between detection
+
+'''
+resources used:
+
+https://www.instructables.com/Vibration-Sensor-SW-420-Raspberry-Pi/
+
+https://www.geeksforgeeks.org/sleep-in-python/
+
+'''
+
+#GPIO SET UP
+
+#configure to your preference. Refer to gpio
+#pin 16/GPIO 23(pi) -> D0(sensor) 
+pin = 23   
+
+#activating broadcom-chip specific pin numbers
+#required to work in python
+GPIO.setmode(GPIO.BCM)
+
+#pin 23 will recieve the input signal coming from sensor
+GPIO.setup(pin, GPIO.IN)
+#END OF GPIO SET UP
+
+def callback(pin):
+    if GPIO.input(pin):
+        print "Movement Detected"
+
+    else:
+        print "Movement Detected"
+
```

Comment:
Although this makes certain that the logic will print the movement, it feels repetitive.